src/i18n/sv.json

```diff
@@ -146,7 +149,7 @@
     "termsLabel": "Jag har bekantat mig med tjänstens registerbeskrivning och stadens sekretesspolicy",
     "terms": "Läs tjänstens <0>registerbeskrivning</0> och stadens <1>sekretesspolicy</1> ",
     "termsDataProtectionLink": "https://www.hel.fi/helsinki/sv/stad-och-forvaltning/information/dataskydd",
-    "termsFileDescriptionLink": "https://www.hel.fi/static/liitteet-2019/Kaupunginkanslia/Rekisteriselosteet/Keha/S%C3%A4hk%C3%B6isten%20asiointipalveluiden%20rekisteri.pdf",
+    "termsFileDescriptionLink": "https://www.hel.fi/static/liitteet-2019/Kaupunginkanslia/Rekisteriselosteet/Keha/Sahkoisten%20asiointipalveluiden%20rekisteri.pdf",
```

Comment:
There's at least https://www.hel.fi/static/liitteet-2019/Kaupunginkanslia/Rekisteriselosteet/Keha/Register%20over%20e-tjanster.pdf for Swedish. Can't find an English version of this document, though.

Reply:
Thx!

There is a note in English version of hel.fi:

"Finnish and Swedish are the only official languages of Finland and, as such, most forms are only available in Finnish and Swedish. 

Click here to read the City's file descriptions in either **Finnish or Swedish**"

Apparently no English pdf exists.

Reply:
Updated the sv.json